tk_toolchain/cmd_line_tools/tk_build_qt_resources/__init__.py

```diff
@@ -99,54 +97,75 @@ def build_res(
     }
 
 
+def build_absolute_path(path_or_file):
+    dir = os.getcwd()
+    combined_path = os.path.normpath(os.path.join(dir, path_or_file))
+
+    return combined_path
+
+
+def run_yaml_commands(yaml_file, uic, rcc):
+    yaml = YAML()
+    yaml_commands = yaml.load(open(build_absolute_path(yaml_file)))
+
+    for command_set in yaml_commands:
+        ui_src = command_set.get("ui_src")
+        if not ui_src:
+            print("'ui_src' is required in yml config")
+            return 1
```

Comment:
What's this? The method's caller does not check the return value. Nor the rest of the method returns integer.